specification/network/resource-manager/Microsoft.Network/stable/2019-07-01/ipTags.json

```diff
@@ -0,0 +1,406 @@
+            "type": "string",
+            "description": "The name of the ipTag."
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/IpTags"
+            },
+            "description": "Parameters supplied to the create or update IpTags operation."
+          },
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Request successful. The operation returns the resulting ipTag resource.",
+            "schema": {
+              "$ref": "#/definitions/IpTags"
+            }
+          },
+          "201": {
+            "description": "Create successful. The operation returns the resulting ipTag resource.",
+            "schema": {
+              "$ref": "#/definitions/IpTags"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "azure-async-operation"
+        },
+        "x-ms-examples": {
+          "CreateOrUpdate_IpTag": {
+            "$ref": "./examples/IpTagsCreate.json"
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "IpTags"
+        ],
+        "operationId": "IpTags_Update",
+        "description": "Updates an ipTag in a specified resource group.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "ipTagName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the ipTag."
+          },
+          {
+            "name": "ipTagsParameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "./network.json#/definitions/TagsObject"
+            },
+            "description": "Parameters supplied to the update ipTag operation."
+          },
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Request successful. The operation returns the resulting ipTag resource.",
+            "schema": {
+              "$ref": "#/definitions/IpTags"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Update_IpTags": {
+            "$ref": "./examples/IpTagsUpdate.json"
+          }
+        },
+        "x-ms-long-running-operation": true
```

Comment:
PATCH to update tags isn't long-running, please remove `x-ms-long-running-operation`

Reply:
Yeah, that's an error we haven't had a chance to fix. But generally, if your method doesn't return anything beside 200 it's not a long-running operation

Reply:
Can I leave this out for consistency across resources?

Reply:
I would say we should ignore consistency in this case since this flag is going to give us wrong code in generated SDKs
@KrisBash, could you please take a look?

Reply:
I agree - if it's a synchronous API that only returns 200, let's remove the x-ms-long-running-operation. If for no other reason, it's confusing. Consistently inconsistent is not desirable :)

Reply:
Done